### DISCUSSION TITLE
University documents keep getting rejected

### DISCUSSION THREAD

**Discussion by SharmaMehak0831** [community/community#146045] [GitHub Education] [answered]
### Hi everyone,

Hi GitHub Education Team

I am a student at KU Leuven, Belgium. I keep trying to upload my documents to get access to GitHub education. I have uploaded my enrollment letter (PDF converted to JPG) and student ID (scanned by Adobe Scan). I have also updated my billing information and set up 2-factor authentication. I cannot reupload the same document again and haven't had my exams yet to get transcripts. A factor in the problem is my university shifted to ID cards with QR codes which can be scanned for verification - it takes you to a website showing the student details. Another thing is that most documents such as the enrolment letter or transcripts can be downloaded as PDFs, which also have QR codes for verification. I think the proofs I have given are sufficient and verifiable, that is why I request support for the same. Thank you.

Best
Mehak

**Comment by AronSoto** [comment, marked as answer]
Hi @SharmaMehak0831 

I had a similar experience when applying for GitHub Education, and I understand how frustrating it can be. Here are a few suggestions based on what worked for me and others:  

1. **Highlight the QR Codes**: Since your student ID and enrollment letter include QR codes for verification, you might want to emphasize this in the "additional notes" or "comments" section when submitting your application. Something like:  
   *"Both my student ID and enrollment letter have QR codes that link to official KU Leuven verification pages."*  
   This helps the reviewer understand the unique setup your university uses for validation.  

2. **Combine Documents If Possible**: If you can’t reupload individual files, consider combining your documents into a single PDF (e.g., enrollment letter + student ID + a short explanatory note) and submitting it via a support ticket or any follow-up link provided. In addition, it is also important to check if you have confirmed your university's institutional email address, in order to be able to make the request.

3. **Contact GitHub Education Support**: If your application is stuck or rejected without clear feedback, you can always reach out to [GitHub Education Support](https://support.github.com/contact/education). Explain your situation, including the lack of traditional transcripts and the use of QR-code-based verification. They’re usually understanding if you provide all the necessary context.  

4. **Be patient and perseverant**: The process can take some time, especially during peak periods when many students submit applications. Keep an eye on your email (including spam folders) in case you receive any communication from GitHub.  

> [!IMPORTANT]
> Finally, if nothing works, you may want to ask your university administration office for the physical ID so that instead of the QR code you can use it on the application.

Good luck and I hope you receive your GitHub Education benefits soon.
